db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+# Model for Race
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+# Model for Skill
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE, related_name='skills')
+
+    def __str__(self):
+        return self.name
+
+# Model for Guild
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
```

Comment:
It's good to use `null=True` for the `description` field in the `Guild` model to allow NULL values in the database. Consider adding `blank=True` if you also want to allow empty strings in forms.